webrtc.html

```diff
@@ -4790,23 +4770,74 @@ <h2>Methods</h2>
               <p>The resulting certificate MUST NOT include information that
               can be linked to a user or <a>user agent</a>. Randomized values
               for distinguished name and serial number SHOULD be used.</p>
-              <p>A <a>user agent</a> MUST reject a call to
-              <code>generateCertificate()</code> with a
-              <code>DOMException</code> of type <code>NotSupportedError</code>
-              if the <var>keygenAlgorithm</var> parameter identifies an
-              algorithm that the <a>user agent</a> cannot or will not use to
-              generate a certificate for <code>RTCPeerConnection</code>.</p>
-              <p>The following values MUST be supported by a <a>user agent</a>:
-              <code>{ name: "<a data-cite=
-              "!WebCryptoAPI#rsassa-pkcs1">RSASSA-PKCS1-v1_5</a>",
-              modulusLength: 2048, publicExponent: new Uint8Array([1, 0, 1]),
-              hash: "SHA-256" }</code>, and <code>{ name: "<a data-cite=
-              "!WebCryptoAPI#ecdsa">ECDSA</a>",
-              namedCurve: "<a data-cite=
-              "!WebCryptoAPI#dfn-NamedCurve">P-256</a>"
-              }</code>.</p>
-              <p class="note">It is expected that a <a>user agent</a> will have
-              a small or even fixed set of values that it will accept.</p>
+
+              <p>When the method is called, the user agent MUST run the following steps:</p>
+              <ol>
+                <li>Let <var>keygenAlgorithm</var> be the first argument to
+                  <code>generateCertificate</code>.</li>
+                <li>Let <var>expires</var> be a <code>DOMTimestamp</code> value
+                  of 2592000.</li>
```

Comment:
If this was seconds it would be 30 days, but DOMTimeStamp is defined in terms of milliseconds. Make it 2592000000.

And perhaps?
```
<div class="note">
  <p>This means the certificate will expire in 30 days from the time of the
  <code>generateCertificate</code> call.</p>
</div>
```

Reply:
ok